deploy/weekly-environment/scripts/benchmarks/kspm_vanilla/data/package_policy_vanilla.json

```diff
@@ -15,50 +15,24 @@
                     "data_stream": {
                         "type": "logs",
                         "dataset": "cloud_security_posture.findings"
-                    },
-                    "release": "ga"
-                }
-            ]
-        },
-        {
-            "type": "cloudbeat/cis_eks",
-            "policy_template": "kspm",
-            "enabled": false,
-            "streams": [
-                {
-                    "enabled": false,
-                    "data_stream": {
-                        "type": "logs",
-                        "dataset": "cloud_security_posture.findings"
-                    },
-                    "release": "ga",
-                    "vars": {
-                        "access_key_id": {
-                            "type": "text"
-                        },
-                        "secret_access_key": {
-                            "type": "text"
-                        },
-                        "session_token": {
-                            "type": "text"
-                        },
-                        "shared_credential_file": {
-                            "type": "text"
-                        },
-                        "credential_profile_name": {
-                            "type": "text"
-                        },
-                        "role_arn": {
-                            "type": "text"
-                        }
                     }
                 }
             ]
         }
     ],
     "package": {
         "name": "cloud_security_posture",
-        "title": "Kubernetes Security Posture Management (KSPM)",
-        "version": "1.1.1"
+        "title": "Security Posture Management (CSPM/KSPM)",
+        "version": "1.2.10"
```

Comment:
this version is not stable, no?

Reply:
As far as I know the broken integration is `1.2.11`, but the integration will auto-upgrade to `1.2.11` after the deployment.
As far as I saw I got findings as usual, although the agent wasn't healthy (on 1.2.11).

Do you think it would be better to deploy @jeniawhite version? `1.2.12-next`